templates/loginMerge.html

```diff
@@ -10,20 +10,20 @@ <h3 class="panel-title">{{ match.name }} ({{ match.email }})</h3>
         </div>
         <div class="panel-body form-signin">
           {% for provider in match.providers %}
-            <a class="button zocial {{ provider.name | lower }}" href="{% url 'scopedallaccess-login' provider.name %}">
-              Zaloguj z {{ provider.name | title }}
+            <a class="button zocial {{ provider.name | lower }}" href="{% url 'social:begin' provider %}?next={% url 'finish_merge_verification' %}">
```

Comment:
What is this next argument for? Can it be potentially manipulated in some way to do weird things?